app/src/main/java/grinnell/appdev/edu/lyles/fragments/BeerFragment.java

```diff
@@ -1,22 +1,117 @@
 package grinnell.appdev.edu.lyles.fragments;
 
+import android.os.AsyncTask;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
-import android.support.v4.app.Fragment;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
 
 import grinnell.appdev.edu.lyles.R;
 
-/**
- * Created by Mattori on 5/9/16.
- */
-public class BeerFragment extends Fragment {
+public class BeerFragment extends AppCompatActivity {
+
+    ListView list;
+    BeersAdaptor adaptor;
+    ArrayList<Beers> beersList;
+
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.beer_layout, container, false);
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.beer_layout);
+
+        list = (ListView) findViewById(R.id.lvBeers);
+        beersList = new ArrayList<Beers>();
+
+        new BeersAsynTask().execute("http://www.cs.grinnell.edu/~birnbaum/appdev/lyles/beer.json");
+
+    }
+
+
+
+    public class BeersAsynTask extends AsyncTask<String, Void, Boolean> {
+
+        HttpURLConnection urlConnection;
+
+        @Override
+        protected Boolean doInBackground(String... params) {
+
+            StringBuilder result = new StringBuilder();
+
+            try {
+                URL url = new URL("http://www.cs.grinnell.edu/~birnbaum/appdev/lyles/beer.json");
```

Comment:
Since we started using the `OkHttp` library, you should switch over to using that.